### PR TITLE
PHPStan setup and fixes

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/ApsMapper/Traits/ProgrammeUtilitiesTrait.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ protected function assertIsProgramme($item)
         }
     }
 
-    protected function formatDateTime(\DateTimeImmutable $dateTimeImmutable = null): string
+    protected function formatDateTime(\DateTimeImmutable $dateTimeImmutable = null): ?string
```
Comment (votemike) [inline src/ApsMapper/Traits/ProgrammeUtilitiesTrait.php]: `use DateTimeImmutable`?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by felipeparaujo** [comment]
test this please

**Comment by felipeparaujo** [comment]
test this please

**Comment by felipeparaujo** [comment]
test this please

**Comment by felipeparaujo** [comment]
test this please

File: script/test [hunk 2]
```diff
@@ -8,5 +8,6 @@ if ! (php --version | grep -q "PHP 7.1.") ; then
     fi
 fi
 
-vendor/bin/phpunit
-vendor/bin/phpcs
+#vendor/bin/phpunit
```
Comment (votemike) [inline script/test]: This doesn't look right...

File: script/test [hunk 3]
```diff
@@ -8,5 +8,6 @@ if ! (php --version | grep -q "PHP 7.1.") ; then
...
+#vendor/bin/phpcs
```
Comment (votemike) [inline script/test]: Uncomment this too?

**Pull request merged**